app/controllers/two_factor_authentication/recovery_code_verification_controller.rb

```diff
@@ -5,6 +5,11 @@ class RecoveryCodeVerificationController < DeviseController
 
     prepend_before_action :authenticate_scope!
 
+    def show
+      @delivery_method = 'recovery-code'
+      @delivery_method
```

Comment:
Thats what I thought, but the controller wasn't returning this value until I split it out on to two lines. Maybe the root cause was something else.